tasks/db_deploy/create_db.py

```diff
@@ -0,0 +1,144 @@
+"""
+Name: create_db.py
+
+Description: Creates the current version on the ORCA database.
+"""
+from shared_db import get_configuration, get_root_connection, logger
+from orca_sql import *
+
+
+def create_orca_database():
+    # Check to see if database exists if it does not create the database.
+    # root_connection = shared_db.get_root_connection(config)
+
+    # TODO: Need to fix this so it is not in a transaction
+    # with root_connection.connect() as conn:
+    #    _create_app_database(conn, config["database"], config["root_user"])
+
+    # Check to see if database exists if it does then perform more checks to
+    # determine the next actions to take otherwise create the database, users,
+    # roles and install the schema
+    pass
+
+
+def create_fresh_orca_install(config):
+    """
+    This task will create the ORCA roles, users, schema, and tables needed
+    by the ORCA application as a fresh install.
+
+    Args:
+        config (Dict): Dictionary with database connection information
+    """
+    # Assume the database has been created at this point. Connect to the ORCA
+    # database as a super user and create the roles, users,  schema, and
+    # objects.
+    root_app_connection = get_root_connection(config, config["database"])
+
+    with root_app_connection.connect() as conn:
+        # Create the roles, schema and user
+        create_app_schema_role_users(conn, config)
+
+        # Change to DBO role and set search path
+        set_search_path_and_role(conn)
+
+        # Create the database objects
+        create_metadata_objects(conn)
+        create_recovery_objects(conn)
+
+        # If everything is good, commit.
+        conn.commit()
+
+
+def create_app_schema_role_users(connection, config):
+    """
+    Creates the ORCA application database schema, users and roles.
+
+    Args:
```

Comment:
Missing config from args.

Reply:
Also recommend previous comments regarding type hints.

Reply:
Fixed and added type hints